src/types/bitcoinTypes.ts

```diff
@@ -1 +1,19 @@
+import { Bytes } from "@polkadot/types";
```

Comment:
If we just require the btcTxId on `execute`, we don't need those involved types.